# Use EasyBuilds CC and CFLAGS
# SEP 4th 2020 by B. Hajgato (UGent)
--- Makefile.org	2019-12-17 17:05:14.000000000 +0100
+++ Makefile	2020-09-04 11:06:23.399740263 +0200
@@ -4,7 +4,8 @@
 VERSION=3.3
 
 # To test strict C ANSI compliance
-CC = gcc -ansi -pedantic
+CC ?= gcc -ansi -pedantic
+CFLAGS = ${EBVARCFLAGS} -ansi -pedantic
 LIB= -lm
 
 # This allow you to write commands like "make PURE=purify demo1"
@@ -26,14 +27,14 @@
 
 # If endianess of your machine is not automatically detected in Misc/common.h
 # you should manually specify here
-CFLAGS += -DLITTLE_ENDIAN
+# CFLAGS += -DLITTLE_ENDIAN
 #CFLAGS += -DBIG_ENDIAN
 
 #######################
 # GENERAL FLAGS FOR GCC
 
 # Optimized code
-CFLAGS += -O6
+#CFLAGS += -O6
 
 # Debug mode with gdb
 #CFLAGS += -g 
@@ -48,7 +49,7 @@
 CFLAGS += -IParser -IStandalone -IMisc -ILibOneChannel -ILibMultiChannel -IEngine -IDatabase
 
 # Flags for GCC
-CFLAGS += -Wall
+#CFLAGS += -Wall
 # CFLAGS += -fhandle-exceptions
 
 # Flags for OS/2
